client/analytics/settings/config.js

```diff
@@ -65,4 +66,36 @@ export const analyticsSettings = applyFilters( SETTINGS_FILTER, [
 		initialValue: wcSettings.wcAdminSettings.woocommerce_excluded_report_order_statuses || [],
 		defaultValue: [ 'pending', 'cancelled', 'failed' ],
 	},
+	{
+		name: 'woocommerce_rebuild_reports_data',
+		label: __( 'Rebuild reports data:', 'wc-admin' ),
+		inputType: 'button',
+		inputText: __( 'Rebuild reports', 'wc-admin' ),
+		helpText: __(
+			'This tool will rebuild all of the information used by the reports. ' +
+				'Data will be processed in the background and may take some time depending on the size of your store.',
+			'wc-admin'
+		),
+		callback: ( resolve, reject, addNotice ) => {
+			const errorMessage = __( 'There was a problem rebuilding your report data.', 'wc-admin' );
+
+			apiFetch( { path: '/wc/v3/system_status/tools/rebuild_stats', method: 'PUT' } )
```

Comment:
This can work for now if it's to involved (I know the needs of this tool might change per the comment above) -- but should we consider implementing a proper action/mutator for this?

Reply:
I think that's probably a smart move long-term.  It may be something we want to wait for https://github.com/woocommerce/wc-admin/issues/1358 until we decide to add in since it will likely change this.

That also makes me think that while we can import/export the store here, I'm not sure the cleanest way of doing this is from a config file and this may require some refactoring in how settings are handled.